common/lib/xmodule/xmodule/modulestore/split_mongo/split.py

```diff
@@ -2684,6 +2684,28 @@ def copy_all_asset_metadata(self, source_course_key, dest_course_key, user_id):
                 # update the index entry if appropriate
                 self._update_head(dest_course_key, index_entry, dest_course_key.branch, new_structure['_id'])
 
+    def force_publish_course(self, course_locator, user_id, commit=False):
```

Comment:
This method actually shouldn't be in this file; it should be in `split_draft.py`